content/PreparedStatement/PreparedStatement.q.md

```diff
@@ -0,0 +1,1709 @@
+# Prepared Statement
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true, init=[commands/PreparedStatement/InsertValuesBerryVikkiIntoStudentTable.sql], filename=[connection.php,student.php]
+
+```php
+// connection.php
```

Comment:
We need to make this `readonly`, we don't want them to use our application to connect any other DB outside of codestop.

Reply:
Also can we make the username `codestop`?